vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/AnalysysTestBase.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.ReactNative.Managed.CodeGen.UnitTests
+{
+  public abstract class AnalysysTestBase
+  {
+    public TestContext TestContext { get; set; }
+
+    protected CancellationTokenSource TokenSource;
+
+    public AnalysysTestBase()
+    {
+      TokenSource = new CancellationTokenSource();
+    }
+
+    protected (CodeAnalyzer, INamedTypeSymbol) AnalyzeModuleFile(string csSnippet)
+    {
+      var csCode = @"
+using System;
+using Microsoft.ReactNative.Managed;
+
+[ReactModule]
+public class TestClass
+{
+" + csSnippet + @"
+}
+";
+
+      return AnalyzeFileAsync(csCode).GetAwaiter().GetResult();
+    }
+
+    protected (CodeAnalyzer, INamedTypeSymbol) AnalyzeFile(string csCode)
+    {
+      return AnalyzeFileAsync(csCode).GetAwaiter().GetResult();
+    }
+
+    protected async Task<(CodeAnalyzer, INamedTypeSymbol)> AnalyzeFileAsync(string csCode)
+    {
+      var codeAnalyzer = new CodeAnalyzer(TokenSource.Token);
+
+      codeAnalyzer.AddSyntaxTree(
+        CSharpSyntaxTree.ParseText(csCode, path: "test.cs", options: new CSharpParseOptions()));
+
+      var references = new List<string>();
+      references.AddRange(Directory.EnumerateFiles(@"C:\Users\dannyvv\.nuget\packages\microsoft.netcore.universalwindowsplatform\6.2.8\ref\uap10.0.15138", "*.dll"));
```

Comment:
fix

Reply:
Sorry this one slipped through from getting bootstrapped.